include/RED4ext/Scripting/Natives/IUpdatableSystem-inl.hpp

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#ifdef RED4EXT_STATIC_LIB
+#include <RED4ext/Scripting/Natives/IUpdatableSystem.hpp>
+#endif
+
+#include <RED4ext/SystemUpdate.hpp>
+
+RED4EXT_INLINE void RED4ext::IUpdatableSystem::OnRegisterUpdates(RED4ext::UpdateRegistrar* aRegistrar)
```

Comment:
I see this is an interface, do we want defaults for it? I would have expected that this is a pure interface or even the game is like this?

Reply:
It has an empty default implementation.